install/v2plugin/README.md

```diff
@@ -43,29 +45,102 @@ To create a plugin from [contiv repo](https://github.com/contiv/netplugin), enab
 make demo-v2plugin
 ```
 
-## Contiv plugin-modes
+## Contiv plugin-roles
 Contiv plugin runs both netplugin and netmaster by default. Contiv v2plugin can be run with only netplugin by setting the plugin_role to slave.
 ```
-docker plugin install contiv/v2plugin:1.0.0-beta.3 iflist=eth1,eth2 plugin_role=slave
+docker plugin install contiv/v2plugin:0.1 iflist=eth1,eth2 plugin_role=slave
 ```
 
-## Contiv plugin workflow
+## Contiv plugin compose workflow (recommended and default for v2plugin)
   1. Etcd cluster should be brought up on the hosts on localhost:2379. If a different port (or Consul) is used, cluster-store option needs to be specified in the plugin install command.
 
+  2. Bringup Docker Swarm-mode
+  ```
+  # on manager node init swarm-mode
+  docker swarm init --advertise-addr 192.168.2.10:2377
+
+  # get the join-token from master node
+  docker swarm join-token worker -q
+
+  # on worker nodes, use the token to join swarm
+  docker swarm join --token SWMTKN-1-4qgg20vkzhc3jhc765k5x0coyriggkdvw1t7fmbiimqguagqr7-8um9goip0d03yqmmrb4c4fh1j 192.168.2.10:2377  
+  ```  
+  3. Install contiv v2plugin
+  ```
+  # on swarm manager node install plugin with 'master' role
+  docker plugin install contiv/v2plugin:0.1 plugin_role=master iflist=<data ifs used for vlan networks>
+  ( allow/grant the install permissions when prompted )
+
+  # on worker nodes, install plugin with 'slave' role
+  docker plugin install contiv/v2plugin:0.1 plugin_role=slave iflist=<data ifs used for vlan networks>
+
+  # to see if the plugin is installed and enabled
+  docker plugin ls
+  # also check is netplugin/netmaster started
+  cat /var/run/contiv/log/plugin_bootup.log
+  ```
+  ```
+  If there are multiple local interfaces you need specify the local IP address to use.
+  docker plugin install contiv/v2plugin:0.1 ctrl_ip=192.168.2.10 control_url=192.168.2.10:9999 iflist=eth2,eth3
```

Comment:
ctrl_ip is the IP passed in as the argument to netplugin for control communication.
control_url is the url passed to netmaster for control communication
Wanted to keep it separate, so that user can configure it as different if there is a usecase